code/game/turfs/simulated/walls.dm

```diff
@@ -86,7 +86,8 @@
 
 	if(rotting)
 		. += "<span class='warning'>There is fungus growing on [src].</span>"
-	. += "<span class='notice'>Using a lit welding tool on this item will allow you to slice through it, eventually removing the outer layer.</span>"
+	if (can_dismantle_with_welder)
```

Comment:
```suggestion
	if(can_dismantle_with_welder)
```

Reply:
already done that, henri beat you too it